docs/resources/aws_cloudwatchlogs_destinations.md

```diff
@@ -0,0 +1,69 @@
+---
+title: About the aws_cloudwatchlogs_destinations Resource
+platform: aws
+---
+
+# aws_cloudwatchlogs_destinations
+
+Use the `aws_cloudwatchlogs_destinations` InSpec audit resource to test properties of the plural resource of AWS Logs Destination.
+
+The AWS::Logs::Destination specifies a CloudWatch Logs destination. A destination encapsulates a physical resource (such as an Amazon Kinesis data stream) and enables you to subscribe that resource to a stream of log events.
+
+## Syntax
+
+Ensure that the destination name exists.
+
+    describe aws_cloudwatchlogs_destinations do
+      it { should exist }
+    end
+
+## Parameters
+
+For additional information, see the [AWS documentation on AWS::Logs::Destination.](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-destination.html).
+
+## Properties
+
+| Property | Description | Field | 
+| --- | --- | --- |
+| destination_names | The name of the destination. | destination_name |
+| target_arns | The Amazon Resource Name (ARN) of the physical target where the log events are delivered (for example, a Kinesis stream). | target_arn |
+| role_arns | A role for impersonation, used when delivering log events to the target. | role_arn |
+| access_policies | An IAM policy document that governs which Amazon Web Services accounts can create subscription filters against this destination. | access_policy |
+| arns | The ARN of this destination. | arn |
+| creation_times | The creation time of the destination, expressed as the number of milliseconds after Jan 1, 1970 00:00:00 UTC. | creation_time |
```

Comment:
Should the property be `creation_time`?

Reply:
@IanMadd Yes it should be creation_time instead of creation_times.